styles/styles.css

```
@@ -185,20 +280,26 @@ flex-grow: 1;
     padding-left: 123px;
 }
 
+.delivery {
+    margin:70px 0;
```

```suggestion
    margin: 70px 0;
```
Не забываем про форматирование кода.